connector-packager/connector_packager/xml_parser.py

```diff
@@ -245,6 +246,11 @@ def parse_file(self, file_to_parse: ConnectorFile) -> bool:
                                   file_to_parse.file_name)
                     return False
 
+            # Set the connector version
+            if 'plugin-version' in child.attrib:
```

Comment:
Should we skip in the case of '0.0.0' as well?

Reply:
I'm not sure there's much of a reason to, 0.0.0 is most of the time (outside of our samples) a mistake when making a submission so it's fine to make it easy to notice. The taco can always be manually renamed after packaging (even after signing), so if for some reason it's desired to not have the version number it can be removed.